### ISSUE TITLE
API: Wipe database

### ISSUE THREAD

**Issue by daleharvey** [pouchdb/pouchdb#2349]
Working on a demo, and realised I really want a way to from the server just wipe all clients, preferably automatically by just deleting the database

Use Case: I set up a replication demo, I test it against a bunch of clients, they are now all in sync, my demo works all dandy, now I dont want to demo the the test data I entered, so I delete the online database, however I then need to manually and carefully need to stop all the clients (if they are syncing, the data gets filled straight back in), then individually delete their data, then delete their master

tl;dr our data persistence is too good
